.toys/ci.rb

```diff
@@ -328,7 +328,8 @@ def run_linkinator dir
   skip_regexes = [
     "\\w+\\.md$",
     "^https://rubygems\\.org/gems/#{dir_without_version}",
-    "^https://cloud\\.google\\.com/ruby/docs/reference/#{dir}/latest$"
+    "^https://cloud\\.google\\.com/ruby/docs/reference/#{dir}/latest$",
+    "^https://rubydoc\\.info/gems/#{dir_without_version}"
```

Comment:
Use `#{dir}` instead of `#{dir_without_version}`. The former is the name of the gem; the latter could be the name of the corresponding wrapper gem. As we discussed in the meeting, the `rubydoc.info` links will generally appear for non-cloud gems whereas `cloud.google.com` links will appear for cloud gems. So you can tell what link we may need to omit by correspondence with the existing omitted `cloud.google.com` link in the previous line—which uses `#{dir}`.

As a corollary, we should do the same thing a few lines down after line 335, using the `cloud.google.com` url to model a `rubydoc.info` url to omit. (What's going on there is that the documentation for a wrapper gem includes links to the documentation for GAPIC gems. We detect that by checking if `dir == dir_without_version`, i.e. that there is no version in the gem name. Then we omit any links to corresponding gems that have a version in the name.)

Reply:
I made changes in both the places.